Update Tedit field names in Lafite

rmkaplan opened the pull request:
Lafite is not in the fuller ms database, so I missed these references in TEDIT-INDENT.

But note also:  there are 2 definitions of the function \GV.PARSERECIPIENTS1, one on lafite/TEDIT-PRIVATEDL and one on internal/MAILCLIENT.

The one on TEDIT-PRIVATEDL references the variable ADDRESSPARSERRDTBL, which is not initialized.

nbriggs commented:
You mean LAFITE-PRIVATEDL.  At the time I created LAFITE-PRIVATEDL I think that ADDRESSPARSERRDTBL was defined somewhere in LAFITE, and my code modified the existing value.  Subsequently, Bill incorporated the private DL syntax into the regular Lafite mail client, and changed the way he did the parsing so it no longer used that read table.

rmkaplan commented:
Do you think it still uses that file?  Maybe it show to obsolete.

> On Feb 15, 2025, at 10:30 AM, Nick Briggs ***@***.***> wrote:
> 
> 
> nbriggs
>  left a comment 
> (Interlisp/medley#2034)
> You mean LAFITE-PRIVATEDL. At the time I created LAFITE-PRIVATEDL I think that ADDRESSPARSERRDTBL was defined somewhere in LAFITE, and my code modified the existing value. Subsequently, Bill incorporated the private DL syntax into the regular Lafite mail client, and changed the way he did the parsing so it no longer used that read table.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
> 
>  <https://github.com/Interlisp/medley/pull/2034#issuecomment-2661037120> <https://github.com/notifications/unsubscribe-auth/AQSTUJPX237RAQNU7EY6FXD2P6BSRAVCNFSM6AAAAABXGVO3RKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRRGAZTOMJSGA>
> 
> nbriggs
>  left a comment 
> (Interlisp/medley#2034)
>  <https://github.com/Interlisp/medley/pull/2034#issuecomment-2661037120>
> You mean LAFITE-PRIVATEDL. At the time I created LAFITE-PRIVATEDL I think that ADDRESSPARSERRDTBL was defined somewhere in LAFITE, and my code modified the existing value. Subsequently, Bill incorporated the private DL syntax into the regular Lafite mail client, and changed the way he did the parsing so it no longer used that read table.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/Interlisp/medley/pull/2034#issuecomment-2661037120>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQSTUJPX237RAQNU7EY6FXD2P6BSRAVCNFSM6AAAAABXGVO3RKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRRGAZTOMJSGA>.
> You are receiving this because you authored the thread.
>

rmkaplan commented:
I removed a few more dependencies on Tedit compile-time declarations.  I think what remains is reference to LINEDESCRIPTOR fields in LAFITE-INDENT (a separate utility contributed by Stan Lanning).  But looking at this, I don't think it would work as seems to be intended.  It is confusing the characters of a selection with the lines of the subset of those characters that are visible in any particular window pane.  It will result in an error if the first or last characters are not visible in the first pane, e.g. because of scrolling.

masinter reviewed (approved):
should move LAFITE-INDENT to obsolete? In the meanwhile, open an issue to fix up the features so they deal with wrapped text which has linefeeds.
